### PR TITLE
Remove unneeded 'using OptionSet = ...'

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Review by CyrusNajmabadi — APPROVED** [review]
w00t!

File: src/Analyzers/CSharp/Analyzers/ConvertNamespace/ConvertToFileScopedNamespaceDiagnosticAnalyzer.cs [hunk 1]
```diff
@@ -10,9 +10,6 @@
 using Microsoft.CodeAnalysis.Options;
 using Microsoft.CodeAnalysis.Text;
 
-#if CODE_STYLE
-using OptionSet = Microsoft.CodeAnalysis.Diagnostics.AnalyzerConfigOptions;
-#endif
```
Comment (Youssef1313) [inline src/Analyzers/CSharp/Analyzers/ConvertNamespace/ConvertToFileScopedNamespaceDiagnosticAnalyzer.cs:13]:
```suggestion
```

File: src/Analyzers/Core/Analyzers/ConvertTypeofToNameof/AbstractConvertTypeOfToNameOfDiagnosticAnalyzer.cs [hunk 2]
```diff
@@ -6,9 +6,6 @@
 using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis.Operations;
 
-#if CODE_STYLE
-using OptionSet = Microsoft.CodeAnalysis.Diagnostics.AnalyzerConfigOptions;
-#endif
```
Comment (Youssef1313) [inline src/Analyzers/Core/Analyzers/ConvertTypeofToNameof/AbstractConvertTypeOfToNameOfDiagnosticAnalyzer.cs:9]:
```suggestion
```

**Review by sharwell — APPROVED** [review]

**Comment by Youssef1313** [comment]
@CyrusNajmabadi This should be ready to merge :)

**Pull request merged**